wget/README.md

```diff
@@ -0,0 +1,51 @@
+
+# `wget`
+
+This task performs wget operation to download file from internet to workspace .
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/v1beta1/wget/wget.yaml
+```
+
+## Parameters
+
+* **mwendler/wget**: `wget` CLI container image to run this task.
```

Comment:
Suggest we rename it to `wget-image` or something that makes it clear that this is the container image to be used. Also we should mention the default value for this paramter

Reply:
Also, looks like this is not actually a parameter that is used? I think it is ok for this to not be a param. The `wget` command does not change frequently so its less likely that someone might want to use a different image

Reply:
Ok, I will just delete this line